proposals/stage-1/CODE_OF_CONDUCT/HANDLING-REPORTS.md

```diff
@@ -0,0 +1,41 @@
+
+## Channels
+* Email: report@openjsf.org.
+* Form: It would be easier on the reportee to have a form template to help them structure their report. The form submission could trigger an email.
+* Repo: openjs/moderation (not provisioned). As commented in PR review, hosting sensitive personal information on GitHub needs due consideration, review, and possibly legal consultation.
+  * A form submission could open an issue in the repo, to allow folks to discuss in context.
+## Confidentiality and record-keeping
+Everything reported and discussed is confidential unless explicitly stated. For any facts to be made public, the reportee’s explicit permission is needed. All reports should be recorded, together with the discussion of it. A private moderation repo would satisfy the need for recordkeeping.
+## Steps/Escalation
+1. A report is filed: This is the first step, mostly defined in Channels above.
+2. The right people are alerted: See Roles.
+3. Contact person assigned: See Roles.
+4. Report is acknowledged: The contact person responds to let the reportee know that the report is being discussed.
+5. Facts and opinions are gathered: Time is allocated to collect information in one place to make sure everyone involved has access.
+6. Facts are discussed: The facts are discussed in context of opinions. Also see **What to look forward to** for guidelines on appropriate response.
+7. An action to be taken is arrived at: The action to be taken is decided by consensus (see section on Consensus process) involving the roles listed in the next section.
+8. Resolution offered: The resolution is offered to the reportee by the contact person. The reportee needs to accept the resolution for the matter to be resolved. If they don’t accept, go back to step 5. If there isn't a clear way to resolve the report, the report can be reviewed by the moderation team representatives from OpenJS projects to ensure that the the action and resolution are appropriate.
+The report is resolved: The report is resolved if:
+* the reportee accepts the action.
+* there is no communication for 21 days on behalf of the original reporter.
+* the Ombudsperson decides that the resolution as it stands is the best possible outcome within the constraints of all involved.
+## Roles
+* CPC: The Cross Project Council must get involved in cases where a report has been escalated by a project or its team reponsible for enforcing their Code of Conduct. (TBD: role of regular and voting members)
+* Moderation team representatives: Each project in the CPC which has a moderation team should have representation here. https://github.com/openjs-foundation can ask these folks if any are willing to also moderate this org.
+* Ombudsperson: This person acts as an impartial intermediary between a reporter and the Foundation. They also serve to mediate between the moderation team representatives.
```

Comment:
Will there be any particular rule to decide on who will be this person? Like they cannot be from the same project, just to avoid any conflict of interest.

Reply:
That's a good issue to raise. We should clarify. For example, my own opinion is that this person should currently not be directly involved with a specific project (OpenJS Foundation is fine), but past history is ok.